changes/18187-ai-generated-backend

```diff
@@ -0,0 +1,7 @@
+Added `/fleet/autofill/policies` endpoint to get autogenerated policy description and resolution for a given SQL query.
```

Comment:
Let's add a note that Fleet will now communicate with https://fleetdm.com (maybe users/customers need to open ports?)